test/onnx/parse/dynamicquantizelinear_2d_test.cpp

```diff
@@ -32,25 +32,31 @@ TEST_CASE(dynamicquantizelinear_2d_test)
     auto x_type = migraphx::shape::float_type;
     auto x      = mm->add_parameter("x", {x_type, x_dims});
 
-    auto l0         = mm->add_literal({0.f});
-    auto x_reshaped = mm->add_instruction(migraphx::make_op("reshape", {{"dims", {12}}}), x);
-    x_reshaped = mm->add_instruction(migraphx::make_op("concat", {{"axis", 0}}), x_reshaped, l0);
+    auto l0 = mm->add_literal({0.f});
 
-    auto q_range = mm->add_literal(
-        migraphx::literal{migraphx::shape{x_type}, {std::numeric_limits<uint8_t>::max()}});
+    std::vector<size_t> axes(x->get_shape().lens().size());
```

Comment:
No because it creates a dynamcquantizelinear Onnx op which resolves into what we decide to parse it into on our end

```

@onnx_test()
def dynamicquantizelinear_2d_test():
    x = helper.make_tensor_value_info('x', TensorProto.FLOAT, [3, 4])
    y = helper.make_tensor_value_info('y', TensorProto.UINT8, [3, 4])
    y_scale = helper.make_tensor_value_info('y_scale', TensorProto.FLOAT, [1])
    y_zero_point = helper.make_tensor_value_info('y_zero_point',
                                                 TensorProto.UINT8, [1])

    node = onnx.helper.make_node(
        'DynamicQuantizeLinear',
        inputs=['x'],
        outputs=['y', 'y_scale', 'y_zero_point'],
    )

    return ([node], [x], [y, y_scale, y_zero_point])

```

From our perspective what we do should be opaque to the onnx operator as long as functionalist is equivalent